pkg/webhook/controlplane/add.go

```diff
@@ -20,22 +26,43 @@ import (
 )
 
 var (
-	logger = log.Log.WithName("azure-controlplane-webhook")
+	logger                           = log.Log.WithName("azure-controlplane-webhook")
+	versionConstraintGreaterEqual198 *semver.Constraints
 )
 
+func init() {
+	var err error
+	versionConstraintGreaterEqual198, err = semver.NewConstraint(">= 1.98")
```

Comment:
Can we have a TODO added here as well and for the other similar PRs in the other extensions?